dist/user_install_script.sh

```diff
@@ -0,0 +1,163 @@
+#!/bin/sh
+
+# if a password was set by decky, this will run when the program closes
+temp_pass_cleanup() {
+  echo $PASS | sudo -S -k passwd -d deck
+}
+
+# if the script is not root yet, get the password and rerun as root
+if (( $EUID != 0 )); then
+    PASS_STATUS=$(passwd -S deck)
+    if [ "$PASS_STATUS" = "" ]; then
+        echo "Deck user not found. Continuing anyway, as it probably just means user is on a non-steamos system."
+    fi
+
+    if [ "${PASS_STATUS:5:2}" = "NP" ]; then # if no password is set
+        if ( zenity --title="Decky Installer" --width=300 --height=200 --question --text="You appear to have not set an admin password.\nDecky can still install by temporarily setting your password to 'Decky!' and continuing, then removing it when the installer finishes\nAre you okay with that?" ); then
+            yes "Decky!" | passwd deck
+            trap temp_pass_cleanup EXIT # make sure password is removed when application closes
+            PASS="Decky!"
+        else exit 1; fi
+    else
+        # get password
+        FINISHED="false"
+        while [ "$FINISHED" != "true" ]; do
+            PASS=$(zenity --title="Decky Installer" --width=300 --height=100 --entry --hide-text --text="Enter your admin password")
```

Comment:
I'd assume more people would know this as a "sudo password" rather than "admin password" as a lot of guides refer to it as the sudo password iirc

Reply:
Or just `sudo/admin` to cover both

Reply:
did `sudo/admin`